cmds/ocm/commands/ocmcmds/common/addhdlrs/options.go

```diff
@@ -8,30 +8,51 @@ import (
 )
 
 type Options struct {
-	Replace bool
+	Replace           bool
+	PreserveSignature bool
 }
 
-var _ ocm.ModificationOption = (*Options)(nil)
+var (
+	_ ocm.ModificationOption        = (*Options)(nil)
+	_ ocm.ElementModificationOption = (*Options)(nil)
+	_ ocm.BlobModificationOption    = (*Options)(nil)
+	_ ocm.TargetElementOption       = (*Options)(nil)
+)
 
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	f := fs.Lookup("replace")
-	if f != nil {
-		if bp := generics.Cast[*bool](f.Value); bp != nil {
-			return
-		}
+	if f == nil {
+		fs.BoolVarP(&o.Replace, "replace", "R", false, "replace existing elements")
```

Comment:
Why do you use capslock for R and P? Is there already a shorthand for lower case given?

Reply:
I think the doc "replace existing elements" needs a touchup to describe when to use this flag, not just what it does

Reply:
there is a more detailed description as part of the command docu.

Reply:
For me this does not replace the help for the option since that is what people will reference initially. @fabianburth wdyt?